PIM/Integration/03_ManageAttributeGroups.md

```diff
@@ -0,0 +1,263 @@
+
+
+
+### Change the order of attributes within a group
+
+The order of attributes within an attribute group in the *ATTRIBUTE GROUPS* tab equals to the order of attributes within a group in the product view. You can change the order to display a certain attribute in a different place in the attribute group when creating or editing a product.   
+The procedure to change the order of attributes within attribute groups and sub-groups is identical. Just select the attribute sub-group instead of the group and follow the steps described below.
+
+#### Prerequisites
+
+- At least one attribute group has been created, see [Create an attribute group](#create-an-attribute-group).
+- At least two attributes have been added to the attribute group, see [Add attributes to a group](#add-attributes-to-a-group).
+
+#### Procedure
+*PIM > Settings > Tab ATTRIBUTE GROUPS*
+
+![Attribute groups](../../Assets/Screenshots/PIM/Settings/AttributeGroups/AttributeGroups.png "[Attribute groups]")
+
+1. Click the attribute group or sub-group where you want to change the order of attributes.   
+  The selected attribute group is highlighted. The attributes assigned to the group are displayed on the right side.
+
+  ![Group attributes](../../Assets/Screenshots/PIM/Settings/AttributeGroups/GroupAttributes.png "[Group attributes]")
+
+2. Click the ![Sort attributes](../../Assets/Icons/Sort02.png "[Sort attributes]") (Sort attributes) button above the list.   
+  The *Assigned Attributes (Sorting Mode)* view is displayed on the right side.
+
+  ![Assigned attributes](../../Assets/Screenshots/PIM/Settings/AttributeGroups/AssignedAttributes.png "[Assigned attributes]")
+
+3. Click and hold the ![Sort](../../Assets/Icons/Sort01.png "[Sort]") (Sort) button to the left of the attribute you want to move to another position in the list.
+
+4. By using drag and drop, move the selected attribute to the desired position in the list.
+
+  > [Info] Repeat the steps **3** and **4** for all attributes whose position in the list you want to change.
+
+5. Click the [SAVE] button in the upper right corner.
+  The new order of attributes within the selected attribute group has been saved. The *Assigned Attributes (Sorting Mode)* view is closed.
+
+
+### Remove attributes from a group
+
+You can remove an attribute from an attribute group if it is no longer in use. Note, that you just remove the attribute from the attribute group but you do not delete the attribute itself. Alternatively, you can [deactivate an attribute](./01_ManageAttributes.md#deactivate-an-attribute) so it is no longer displayed in the product view. The procedure to remove attributes from attribute groups or sub-groups is identical. Just select the attribute sub-group instead of the group and follow the steps described below.
```

Comment:
```suggestion
You can remove an attribute from an attribute group if it is no longer in use. Note that you just remove the attribute from the attribute group but you do not delete the attribute itself. Alternatively, you can [deactivate an attribute](./01_ManageAttributes.md#deactivate-an-attribute) so it is no longer displayed in the product view. The procedure to remove attributes from attribute groups or sub-groups is identical. Just select the attribute sub-group instead of the group and follow the steps described below.
```